components/provider-selector.tsx

```diff
@@ -1,3 +1,43 @@
+import styled from "@emotion/styled";
+import Image from "next/image";
+
+const Bar = styled.div`
+  box-sizing: border-box;
+  width: 936px;
+  height: min-content;
+  flex-direction: row;
+  justify-content: flex-start;
+  align-items: center;
+  padding: 16px 16px 16px 16px;
+  background-color: #eeeeee;
+  overflow: hidden;
+  align-content: center;
+  flex-wrap: nowrap;
+  gap: 12;
```

Comment:
```suggestion
  gap: 12px;
```

gap을 px단위로 주어야 적용됩니다.